{"uuid": "2a6c0455-2268-46a8-a1e1-c2d42b172b8d", "counter": 5, "stdout": "ERROR! Unexpected Exception, this is probably a bug: unhashable type: 'AnsibleMapping'", "start_line": 3, "end_line": 4, "runner_ident": "e6343c67-fc64-4265-bde9-faf7625fb82a", "event": "error", "pid": 43, "created": "2020-11-08T11:10:15.141791", "parent_uuid": "fb1bbd4e-c9dd-4c0b-8b63-ce262a445ef0", "event_data": {"playbook": "playbook.yml", "playbook_uuid": "fb1bbd4e-c9dd-4c0b-8b63-ce262a445ef0", "error": true, "uuid": "2a6c0455-2268-46a8-a1e1-c2d42b172b8d"}}